### PR TITLE
feature/ISDK-984: Update package

### PR DESCRIPTION
**Pull request by GraceFirework**
Adding reference to GoogleAds IMA dependency

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -9,13 +9,23 @@ let package = Package(
...
     ],
     dependencies: [
+        .package(
+            url: "https://github.com/loopsocial/GoogleAds-IMA-iOS-SDK.git",
+            .upToNextMajor(from: "3.16.3")
+        )
     ],
     targets: [
         .binaryTarget(name: "FireworkVideoGIMASupport",
                       url: "https://github.com/loopsocial/firework_ios_sdk_google_ima_support/releases/download/v0.2.0/FireworkVideoGIMASupport-v0.2.0.xcframework.zip",
```
Comment (spookyd) [inline Package.swift]: You will need to bump the version so that we can make a new release. Simply update `v0.2.0` instances to `v0.3.0`

Comment (GraceFirework) [reply Package.swift]: thanks, updated

**Review by spookyd — COMMENTED** [review]
Also need to add the `FireworkVideoGIMASupportDependencies` directory under Sources/ with a dummy.swift file. 

Forgot to do this myself and dealing with a botched release :(

**Comment by GraceFirework** [comment]
> Also need to add the `FireworkVideoGIMASupportDependencies` directory under Sources/ with a dummy.swift file.
> 
> Forgot to do this myself and dealing with a botched release :(

I was just wondering about that, thanks updated

**Review by spookyd — APPROVED** [review]

**Pull request merged**